decoders/gstreamer.py

```diff
@@ -70,3 +70,13 @@ class FluendoH265Gst010Decoder(GStreamer010):
     description = "Fluendo H.265 software decoder for GStreamer 0.10"
     codec = Codec.H265
     decoder_element = ' h265parse ! fluh265dec '
+
+
+@register_decoder
+class FluendoH265Gst10Decoder(GStreamer10):
```

Comment:
The name of this class is duplicated. The CI correctly complained about it with `decoders/gstreamer.py:76:0: E0102: class already defined line 58 (function-redefined)` :)

https://dev.azure.com/fluendo/Fluxion/_build/results?buildId=13&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=5caf77c8-9b10-50ef-b5c7-ca89c63e1c86

Reply:
I was testing the CI :wink: